### ISSUE TITLE
SyntaxError: Unexpected token{

### ISSUE THREAD

**Issue by hfpevc** [pishangujeniya/instagram-helper#61]
**Describe the bug**
Uncaught SyntaxError: Unexpected token { VM:364:6

Following the instructions for "How to delete/unsend all messages?" I am able to follow the steps up until "enter" to run the code.

**To Reproduce**
Following the instructions for "How to delete/unsend all messages?" I am able to follow the steps up until "enter" to run the code.

1. I extra the file, and create an empty folder. I open the special chrome, log into IG and copy and paste the chat thread id to a notepad.
2. I then paste the  InstagramHelper.js file contents into conseole tab. I do not press "enter" here. 
3. The last step is to copy and paste the following code and then press "enter"
"  var ig  = new InstagramHelper();
  ig.startUnsending("your_chat_thread_Id",10);"
4. I copy and paste it, then paste my "you_chat_thread_Id", then remove the " quotations around it, then press "enter" and I get the syntaxerror.

**Expected behavior**

**Screenshots**
NA

**Desktop (please complete the following information):**
 - OS: PC
 - Browser chrome
 - Version 96.0.4664.110

**Additional context**
i tried this with a different "your_thread_Id" and I still get a syntaxerror. Here is the new variable I tried it with. If I leave the " quotations around the "your_thread_Id" then I get VM420:35 as the syntax error. If I take the quotations out, then I get VM421:35 as the syntax error

**Comment by pishangujeniya** [comment]
Please attach screenshots of the red errors you getting.

**Comment by hfpevc** [comment]
Please see attached.

![1 syntaxerror VM80 35](https://user-images.githubusercontent.com/96595663/147315205-8c5940d7-fa07-4fbc-86f1-c0bc38c68647.jpg)
![2 syntaxerror VM80 35](https://user-images.githubusercontent.com/96595663/147315207-8f916d49-ed73-4f54-9c32-ed0deb11fa1c.jpg)
![3 syntaxerror VM80 35](https://user-images.githubusercontent.com/96595663/147315209-c33c82ff-a9cc-44d9-a28f-5fcf65fe4ff4.jpg)
![4 syntaxerror VM80 35](https://user-images.githubusercontent.com/96595663/147315210-a2b957cf-960e-4860-8d51-e9a33b785aae.jpg)
![5 syntaxerror VM80 35](https://user-images.githubusercontent.com/96595663/147315211-8d4ec396-ca92-4a0e-999c-9cb654032ea0.jpg)
![6 syntaxerror VM80 35](https://user-images.githubusercontent.com/96595663/147315212-76d61323-af9a-4481-a7b7-74d853e3ffae.jpg)
![7 syntaxerror VM80 35](https://user-images.githubusercontent.com/96595663/147315213-3a10509f-95cc-4c00-9a60-eeea26a8b132.jpg)

**Comment by pishangujeniya** [comment]
You need to press Enter after you paste the whole code from file.

And then after you need to run that two lines of code passing chat thread id in double quotes (not without number)

**Comment by hfpevc** [comment]
When I press Enter after I paste the whole code from the file I get another error (picture attached)
![Uncaught Syntaxerror again](https://user-images.githubusercontent.com/96595663/147813303-58a84abc-ca34-4aa3-8e88-fe26a3079adf.jpg)
.

**Comment by pishangujeniya** [comment]
Can you put a screenshot of the error which mentions _"Unexpected token = "_ after that at the most right side will be written underlines words, press on that and take a screenshot and attach here.

**Comment by hfpevc** [comment]
Ok, I took a picture. Line 35
![unexpected token line 35](https://user-images.githubusercontent.com/96595663/147845306-30d6cdce-8125-4469-86c0-0e39195baaf7.jpg)

**Comment by pishangujeniya** [comment]
Is your browser is updated with latest version?

**Comment by hfpevc** [comment]
Thanks, I was able to relink the special chrome to an updated version which solved one piece of the error. Below, I am getting another "looping" error after I paste the second piece of the code. I did a quick screen shot before the error message fills up the screen and I have to eventually exit and restart the chrome broswer.
![getting message loop](https://user-images.githubusercontent.com/96595663/147847743-b797c277-6851-4f4e-8399-0c87a3ceac30.jpg)
